File: app/src/main/java/io/github/freewebmovement/igniter/persistence/database/AppDatabase.java
package io.github.freewebmovement.igniter.persistence.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;
@Database(entities = {Server.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
//    public abstract EnabledPackageDao enabledPackageDao ();
    public abstract ServerDao serverDao();
//    public abstract UserDao userDao();
}

File: app/src/main/java/io/github/freewebmovement/igniter/persistence/database/AppDatabase.kt
package io.github.freewebmovement.igniter.persistence.database

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [Server::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    //    abstract fun enabledPackageDao(): EnabledPackageDao
    abstract fun serverDao(): ServerDao
    //    abstract fun userDao(): UserDao
}
